extract-metadata.py

```diff
@@ -0,0 +1,84 @@
+from dandi.dandiapi import DandiAPIClient
+import pandas as pd
+from pathlib import Path
+
+modalities = {'oct': 'PS-OCT',
+              'df': 'Dark Field Microscopy',
+              'xpct': 'HiP-CT',
+              'dwi': 'DWI',
+              'fluo': 'LSM',
+              'photo': 'Blockface photo'}
+
+# Create dataframe of all assets across all datasets
+def extract_assets():
+    client = DandiAPIClient("https://api.lincbrain.org/api")
+    client.dandi_authenticate()
+    dandisets = client.get_dandisets()
+
+    print(f"Processing {sum(1 for _ in dandisets)} Dandisets on lincbrain.org")
+
+    df = pd.DataFrame(columns=["Dandiset",
+                            "Version",
+                            "Subject", 
+                            "Modality",
+                            "Path",
+                            "Filename",
+                            "Extension",
+                            "Directory", # Top-level directory (e.g. source data, raw data, derivatives)
+                            'Size (bytes)'])
+
+    for dandiset in dandisets:
+        latest_dandiset = dandiset.for_version('draft')
```

Comment:
Just want to understand this line -- are you only referencing the `draft` version?

Reply:
Yes, exactly.  Just want to retrieve the latest data.